care/facility/api/serializers/patient.py

```diff
@@ -210,6 +210,15 @@ def validate_year_of_birth(self, value):
             raise serializers.ValidationError(msg)
         return value
 
+    def validate_name(self, value):
+        if value is None:
+            raise serializers.ValidationError("Patient name is required.")
+        if not value.strip():
+            raise serializers.ValidationError("Patient name cannot be empty.")
+        if any(char.isdigit() for char in value):
```

Comment:
This will prevent numbers, but what about dots, commas, underscores? I think we should use regex here.

Reply:
My thoughts exactly but then i decided to stick with what the issue title was haha